ska_helpers/git_helpers.py

```diff
@@ -38,12 +39,12 @@ def make_git_repo_safe(path: str | Path) -> None:
 
     with chdir(path):
         # Run a lightweight command which will fail for an unsafe repo
-        proc = subprocess.run(["git", "rev-parse"], capture_output=True)
+        proc = subprocess_win.run(["git", "rev-parse"], capture_output=True)
```

Comment:
It seems using gitpython's rev_parse here should be faster and more Pythonic.

Reply:
I'm fine with using `gitpython`. I originally went with subprocess because there was this issue with `gitpython` giving a completely obscure exception I found difficult to use. And I didn't know about the `rev_parse` interface or the fact that you can update the global config with a `Repo` object.

Reply:
> I originally went with subprocess because there was this issue with gitpython giving a completely obscure exception I found difficult to use. 

Yep. It seemed to be returning more well-behaved exceptions for status() and rev_parse(), but I've forgotten what we were doing that was throwing the obscure exceptions with the content of the help for`git diff --cached` and complaints that `--cached` wasn't an option...